docs/intellicode-visual-studio-code.md

```diff
@@ -16,24 +16,43 @@ The [IntelliCode extension for Visual Studio Code](https://marketplace.visualstu
 
 ![IntelliCode for Python in Visual Studio Code](media/python-intellicode.gif)
 
-## Supported languages
-
 The supported languages for AI-assisted IntelliSense completions are:
 
 - Java
 - JavaScript and TypeScript
 - Python
 - SQL
 
-## Prerequisites
 
-To install the extension, you need Visual Studio Code October 2018 Release 1.29 or later.
+### Prerequisites
+
+The extension is supported by Visual Studio Code October 2018 Release 1.29 or later.
+
+AI-assisted IntelliSense for Java requires Java 8 Update 151 or higher. 
+## Whole-line autocomplete preview
```

Comment:
I believe we decided on the name to be `whole line completions`

Reply:
That's what we decided at VS22 launch time with marketing.
Since then, there's been two clarifications
1) Our copywriter says that whole must always have a dash with line (Brick George has the full details on why this grammar is important)
2) Autocomplete is more immediately relateable